releasenotes/notes/channel-module-deprecations-in-pulse-ffc384b325b077f1.yaml

```
@@ -1,9 +1,10 @@
 ---
 upgrade:
   - |
-    Channel ``buffer`` option was deprecated in Terra 0.11.0 and has now been
-    removed. To add a delay on a channel, specify it explicitly in your
-    Schedule with a Delay::
+    The pulse ``buffer`` option found in Channel and Schedule was deprecated
```

Only one minor thing would be to link to the class documentation when referencing them:

```suggestion
    The pulse ``buffer`` option found in :class:`qiskit.pulse.Channel` and :class:`qiskit.pulse.Schedule` was deprecated
```